server/src/api/modules/groups/services/FetchGroupDetailsService.ts

```diff
@@ -29,7 +34,8 @@ async function fetchGroupDetails(id: number): Promise<GroupDetails> {
     // Sort the members list by role
     memberships: group.memberships.sort(
       (a, b) => priorities.indexOf(b.role) - priorities.indexOf(a.role) || a.role.localeCompare(b.role)
```

Comment:
Shouldn't we be sorting the members list here? If roleOrders exist: use those, otherwise fallback to the current behaviour (using `priorities`)

Reply:
updated with new util function